docs/titan-reference/button-reference.md

```diff
@@ -0,0 +1,485 @@
+ Moves to the next cue in the connected cue list using programmed fade times. Labelled \<Next Cue\> or \<Cue +\> on some consoles. See [Running a cue list](../cue-lists/cue-list-playback.md/#running-a-cue-list).
+
+ --- 
+## \<Next Time\> 
+ Lets you edit timing information for the next cue in a connected cue list. See [Edit times of a running cue list](../cue-lists/editing-cue-lists.md/#edit-times-of-a-running-cue-list).
+
+ --- 
+## \<Not\> 
+ Used when selecting ranges of items. See [Selecting fixtures and dimmers by number](../controlling-fixtures.md/#selecting-fixtures-and-dimmers-by-number-channel).
+
+ Example: \<Fixture\> 1 \<Through\> 5 \<not\> 3 selects fixtures 1, 2, 4, 5
+
+ --- 
+## \<Odd/Even\> 
+ Replaced by \<All\>.  Used to start and end odd/even selection from a range of selected fixtures. See [Selecting fixtures using a pattern](../controlling-fixtures.md/#selecting-fixtures-using-a-pattern).
+
+ --- 
+## \<Off\> 
+ Used to deactivate specific attribute values from fixtures or in playbacks. Attributes set to Off can later be restored to their previous values. See [Deactivate attributes using Off](../cues/editing-cues.md/#deactivate-attributes-from-cues-using-off).
+
+ --- 
+## \<Open\> 
+ (D9 only) Open workspace window, like double pressing View. \
+ (Quartz, Titan Mobile) Show details about item, or double press to open workspace window, see \<Open/View\> below.
+
+ --- 
+## \<Open/View\> 
+(Sapphire Touch, Arena, Tiger Touch only) \
+ Followed by **Select** button to show details about item, or double press to open workspace window. See [Palettes](../palettes/editing-palettes.md/#viewing-and-editing-the-content-of-palettes) and [Cues](../cues/editing-cues.md/#playback-and-cue-view), or [Opening Workspace Windows](../titan-basics.md/workspace-windows/#opening-and-positioning-workspace-windows).
+
+ --- 
+## \<Options\> 
+ Followed by **Select** button, shows options screen for item. Usually used with playbacks but can set options for other types of item. See [Playback Options](../cues/playback-options.md/)
+
+ --- 
+## \<Page +\> 
+ Move to next page in the fader bank (consoles with multiple fader banks may have several of these buttons). See also \<Go Page\>. See [Changing playback pages](../cues/cue-playback.md/#changing-playback-pages).
+
+ --- 
+## \<Page -\> 
+ Move to previous page in the fader bank (consoles with multiple fader banks may have several of these buttons). See also \<Go Page\>. See [Changing playback pages](../cues/cue-playback.md/#changing-playback-pages).
+
+ --- 
+## \<Palette\> 
+ Opens the Palette menu which is used when recalling palettes by number. See [Recalling palettes](../palettes/using-palettes.md/#recalling-from-the-keypad).
+
+ --- 
+## \<Patch\> 
+ Opens the Patch menu. See [Patching new fixtures or dimmers](../patching/patching-new-fixtures-or-dimmers.md/).
+
+ --- 
+## \<Playback\> 
+ (D9) (Arena) Opens the Playbacks workspace window showing touch buttons for additional playback storage. 
```

Comment:
So on the D9 this is a currently-unused "noun" key that will feature in syntax in the future. I believe its only function at present is `<View> <Playback>`. Pressing it without View I believe has no action.

On the Arena it's a window shortcut key as you say.